File: cs50x/sentimental/mario/mario.py
from cs50 import get_int

while True:
    height = get_int("Height: ")
    if height > 0 and height < 9:
        break

for rows in range(0, height):
    for spaces in range(1, height - rows):
        print(" ", end="")
    for lHashes in range(height - rows, height + 1):
        print("#", end="")
    print("  ", end="")
    for rHashes in range(height - rows, height + 1):
        print("#", end="")
    print()

File: cs50x/mario/mario (less)/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int height;
    // Prompt for height
    do 
    {
        height = get_int("Height: "); 
    }
    // Reject numbers other than 1-8
    while (height < 1 | height > 8);
   
    // Print pyramid
    for (int row = 0; row < height; row++)
    {
        // Spaces
        for (int spaces = row - height; spaces < -1; spaces++) 
        {
            printf(" ");
        }
        // Hashes
        for (int hashes = 0; hashes < row + 1; hashes++)
        {
            printf("#");
        }
        
        printf("\n");
    }
}
